tests/rbd/test_rbd_snapshot_rename.py

```diff
@@ -0,0 +1,88 @@
+from tests.rbd.exceptions import RbdBaseException
+from tests.rbd.rbd_utils import Rbd
+from utility.log import Log
+
+log = Log(__name__)
+
+
+def run(**kw):
+    """Verification of snap and clone on an imported image.
+
+    This module verifies snapshot rename operations on an image and its clone
+
+    Args:
+        kw: test data
+
+    Returns:
+        int: The return value. 0 for success, 1 otherwise
+
+    Test case covered -
+    CEPH-9833 - verification rename snap on an image and its clone
+    Pre-requisites :
+    1. Cluster must be up and running with capacity to create pool
+       (At least with 64 pgs)
+    2. We need atleast one client node with ceph-common package,
+       conf and keyring files
+
+    Test Case Flow:
+    1. Create a pool
+    2. Create an image
+    3. Create multiple snapshots of this image
+    4. Rename each of the snapshots
+    5. Create a clone of this image
+    6. Create multiple snapshots of the cloned image
+    7. Rename each of the above snapshots
+    """
+    log.info("Running rename snapshots on image and its clone")
+    rbd = Rbd(**kw)
+    pool = rbd.random_string()
+    image = rbd.random_string()
+    snap = rbd.random_string()
+    new_snap = rbd.random_string()
+    dir_name = rbd.random_string()
+    clone = rbd.random_string()
+    size = "10G"
+
+    rbd.exec_cmd(cmd="mkdir {}".format(dir_name))
+    try:
+        if not rbd.create_pool(poolname=pool):
+            # create pool does not catch exceptions, it returns true/false.
+            # so we are returning 1 instead of raising exception
+            return 1
```

Comment:
[Optional]
Can we please avoid the concerns in comments ?
(With discussion in review meeting we planned to use new initial config method in rbd_utils)

Reply:
Done.